owncloudApp/src/main/res/values/strings.xml

```diff
@@ -727,20 +727,24 @@
     <string name="release_notes_proceed">Proceed</string>
     <string name="release_notes_icon">Release note icon</string>
 
-    <string name="release_notes_4_2_0_title_1">Opening files from outside the app using links</string>
-    <string name="release_notes_4_2_0_subtitle_1">A link that points to the app will open the app and then, resolve the file to preview it directly</string>
-    <string name="release_notes_4_2_0_title_2">"Share to" in oCIS accounts allows to upload to any space</string>
-    <string name="release_notes_4_2_0_subtitle_2">Shared files from other apps can be uploaded to any space and not only the Personal one in oCIS accounts</string>
-    <string name="release_notes_4_2_0_title_3">"Apply to all" for conflicts</string>
-    <string name="release_notes_4_2_0_subtitle_3">New dialog to fix all conflicts at once</string>
-    <string name="release_notes_4_2_0_title_4">Auto-refresh when a file is uploaded</string>
-    <string name="release_notes_4_2_0_subtitle_4">The file list will be now refreshed automatically while uploading to the target folder</string>
-    <string name="release_notes_4_2_0_title_5">Download log files</string>
-    <string name="release_notes_4_2_0_subtitle_5">New option to download a log file to the Downloads folder of the device</string>
-    <string name="release_notes_4_2_0_title_6">Minor bugfixes</string>
-    <string name="release_notes_4_2_0_subtitle_6">Some bugs were fixed to improve experience in the app</string>
+    <string name="release_notes_4_2_0_title_1">Latest version with support for Android 6.0</string>
+    <string name="release_notes_4_2_0_subtitle_1">The 4.2.0 version of the app will be the last one that supports Android 6.0</string>
```

Comment:
Don't mention the release number (4.2.0), it could be confusing for users that don't track the releases. Just `Current release will be the last...`